client/src/services/auth.tsx

```diff
@@ -47,7 +43,7 @@ export async function logout() {
   try {
     store.setState({ status: 'unauthenticated' });
     await router.invalidate(); // this will cause redirect to /login
```

Comment:
Move this below the logout mutation. We noticed a bug in gs1 related to this ordering.